<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget {
                                 background-image: url(:/res/background.png);

                                 }
QLabel{
border-radius:4px;
background-color:rgba(60, 60, 60, 100);
color:rgb(255, 255, 255);
}
color:rgb(255, 255, 255);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>801</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>287</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblDate">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color:rgb(255, 255, 255);
background-color:rgba(0, 0, 0, 0)</string>
        </property>
        <property name="text">
         <string>2022/08/23/星期六</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
color:rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="lblPic">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="main.qrc">:/res/type/Qing.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>261</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblwendu">
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;Microsoft YaHei UI&quot;;
color:rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>32℃</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lblCity">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Microsoft YaHei UI&quot;;
color:rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>西安</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblType">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>晴天</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>76</width>
           <height>43</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="lblHL">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>19-35℃</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="lblGanMao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 170, 255);
color:rgb(255, 255, 255);
border-radius:20px;
QLabel{

background-color:rgba(0,0, 0,0);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_6" native="true">
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="main.qrc">:/res/wind.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblfx">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>51</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>风向</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblfs">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>30</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>风速</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_7" native="true">
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="main.qrc">:/res/pm25.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblpm25">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>52</y>
           <width>46</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>PM2.5</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>11</y>
           <width>46</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>PM2.5</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_8" native="true">
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="main.qrc">:/res/humidity.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblshidu">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>30</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>湿度</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>30</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>湿度</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_9" native="true">
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="main.qrc">:/res/aqi.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblquality">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>52</y>
           <width>60</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>空气质量</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>11</y>
           <width>60</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>空气质量</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbldate1">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string>昨天</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate2">
        <property name="text">
         <string>今天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate3">
        <property name="text">
         <string>周一</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate4">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string>周二</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate5">
        <property name="text">
         <string>周三</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblType1">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="main.qrc">:/res/type/BaoXue.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblType2">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="main.qrc">:/res/type/DaXue.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblType3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="main.qrc">:/res/type/QiangShaChenBao.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblType4">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>res/type/ZhenYu.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblType5">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="main.qrc">:/res/type/ZhongYu.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>410</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblQ1">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string>优</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQ2">
        <property name="text">
         <string>良</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQ3">
        <property name="text">
         <string>轻度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQ4">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string>重度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQ5">
        <property name="text">
         <string>严重</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_13" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lblfx1">
        <property name="text">
         <string>东北风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblfx2">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string>东风</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblfx3">
        <property name="text">
         <string>西北风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblfx4">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string>西南风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblfx5">
        <property name="text">
         <string>西风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_14" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lbldate01">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string>1.1</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate02">
        <property name="text">
         <string>1.2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate03">
        <property name="text">
         <string>1.3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate04">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string>1.4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbldate05">
        <property name="text">
         <string>1.5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_15" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>230</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
                   
text-align: center;
                          
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lblwea01">
        <property name="styleSheet">
         <string notr="true">
width: 5px;height:5px;</string>
        </property>
        <property name="text">
         <string>大雨</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblwea02">
        <property name="text">
         <string>中雨</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblwea03">
        <property name="text">
         <string>龙卷风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblwea04">
        <property name="styleSheet">
         <string notr="true">
                             width: 5px;
                             height:5px;
text-align: center;</string>
        </property>
        <property name="text">
         <string>阴天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblwea05">
        <property name="text">
         <string>小雨</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lblHcurve">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblLcurve">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
